service/resource/legacy/resource.go

```diff
@@ -821,91 +821,94 @@ func (s *Resource) processCluster(cluster awstpr.CustomObject) error {
 		return microerror.Maskf(executionFailedError, fmt.Sprintf("could not create ingress load balancer: '%#v'", err))
 	}
 
-	// Create a launch configuration for the worker nodes.
-	lcInput := launchConfigurationInput{
-		clients:             clients,
-		cluster:             cluster,
-		clusterKeys:         clusterKeys,
-		tlsAssets:           tlsAssets,
-		bucket:              bucket,
-		securityGroup:       workersSecurityGroup,
-		subnet:              publicSubnet,
-		instanceProfileName: workerPolicy.GetName(),
-		prefix:              prefixWorker,
-		ebsStorage:          true,
-	}
-
-	// For new clusters don't assign public IPs and use the private subnet.
-	if key.HasClusterVersion(cluster) {
-		lcInput.associatePublicIP = false
-		lcInput.subnet = privateSubnet
-	} else {
-		lcInput.associatePublicIP = true
-		lcInput.subnet = publicSubnet
-	}
-
-	// An EC2 Keypair is needed for legacy clusters. New clusters provide SSH keys via cloud config.
-	if !key.HasClusterVersion(cluster) {
-		lcInput.keypairName = key.ClusterID(cluster)
-	}
+	// During Cloud Formation migration this logic is only needed for non CF clusters.
+	if !key.UseCloudFormation(cluster) {
```

Comment:
The workers launch config and ASG logic is disabled and provided by the CF resource. We will expand this as we migrate more resources to the CF templates.